src/main/target/AOCODARCF4V3/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
-target_stm32f405xg(AOCODARCF4V3)
+target_stm32f405xg(AOCODARCF4V3_SDCard)
+target_stm32f405xg(AOCODARCF4V3_Flash)
```

Comment:
Please use AOCODARCF4V3 for default (flash version)

We have company naming rule, default for flash version, which is:
- AOCODARCF4V3   //Flash version
- AOCODARCF4V3_SD  // SDCard version

Then we need to handle AOCODARCF4V3_SD macron only.,